src/actionSelector.ts

```diff
@@ -75,21 +75,21 @@ export class ActionSelector {
     return undefined;
   }
 
-  private shouldSelectAccept(swap: Swap) {
+  private shouldSelectAccept(swap: Swap): Promise<boolean> {
```

Comment:
❓ is that really needed? I actually had that at some point but then realise that it would be fine to just return booleans.

Reply:
Well, the interface of `RateEvaluationService` is defined to return `Promise`. I guess that was done because potential implementations might want to do async stuff to evaluate. If that is not needed, then we can remove the Promise here.

We added it so that TS actually gives us errors for the return paths where it was not honored.

Reply:
I agree with everything said but I then decided to remove it. I guess it's a question of taste.